packages/ui/src/components/SideMenu/SideMenu.tsx

```diff
@@ -23,6 +24,7 @@ export interface SideMenuProps {
   header?: React.ReactNode;
   footer?: JSX.Element;
   children?: React.ReactNode;
+  trigger?: React.RefObject<HTMLButtonElement>;
```

Comment:
Could maybe do with a comment explaining what it is doing and what it references. Seems to read from the ref over in `AdminLayout` to control focus upon the overlay disappearing right?